softwareComponents/rofiHalInc/include/rofi_hal.hpp

```diff
@@ -2,6 +2,9 @@
 #include <functional>
 #include <memory>
 
+#define TCB_SPAN_NAMESPACE_NAME std
+#include <span.hpp>  // as soon as rofiHalEsp32 supports C++20 replace this with standard span
```

Comment:
Tohle nemůžeš udělat, protože rofiHalInc se používá i na desktop, kde c++20 je...

Reply:
Pokud chceš, můžeš to obalit do `ifdef` maker

Reply:
esp32 idf will be shortly upgraded to version supporting c++20 so this won't be needed anymore. see https://github.com/paradise-fi/RoFI/pull/255